intro-HPC/ch_fine_tuning_job_specifications.tex

```diff
@@ -104,16 +104,20 @@ \section{Specifying memory requirements}
 amount of memory an application will use during execution is often non-trivial,
 especially when one uses third-party software.
 
-The ``eat\_mem'' application just consumes and then releases memory, for the
-purpose of this test. It has one parameter, the amount of gigabytes of memory
-which needs to be allocated.
-
-First compile the program on your machine and then test it for 1 GB:
-\begin{prompt}
-%\shellcmd{gcc -o eat\_mem eat\_mem.c}%
-%\shellcmd{./eat\_mem 1}%
-Consuming 1 gigabyte of memory.
-\end{prompt}
+\ifgent
+  %% Since we don't have monitor at Gent, there is no real point in the eat_mem section
+\else
+  The ``eat\_mem'' application in the HPC examples directory just consumes and then releases memory, for the
+  purpose of this test. It has one parameter, the amount of gigabytes of memory
+  which needs to be allocated.
+  
+  First compile the program on your machine and then test it for 1 GB:
+  \begin{prompt}
+  %\shellcmd{gcc -o eat\_mem eat\_mem.c}%
+  %\shellcmd{./eat\_mem 1}%
+  Consuming 1 gigabyte of memory.
```

Comment:
Since this is often the subject of debate, I will add this as an issue. To be tackled in subsequent PR.